src/tests/ftest/harness/providers.yaml

```
@@ -0,0 +1,21 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+timeout: 120
+server_config:
+  provider_mux: !mux
+    default:
+      provider: None
```

Note: this is read in by python as `"None"` (str) and not the `None` object which results in this variant being skipped.  If we want a test case that ends up using the default provider then `null` should be used:
```suggestion
      provider: null
```